apps/aevm/src/aevm_eeevm.erl

```diff
@@ -1183,11 +1183,7 @@ loop(CP, StateIn) ->
                     %%                       + 0 otherwise
                     {Us0, State1} = pop(State0),
                     State2 = aevm_eeevm_state:set_selfdestruct(Us0, State1),
-                    spend_mem_gas(State, State2);
-                _ ->
-                    error({opcode_not_implemented,
-                      lists:flatten(
-                        io_lib:format("~2.16.0B",[OP]))})
```

Comment:
Doesn't it make more sense to crash with an `opcode_not_implemented` than with a `case_clause`?

Reply:
This may become unapplicable depending on how I address [comment in other PR](https://github.com/aeternity/epoch/pull/1810#discussion_r235635782). Let us re-check this in the new version of PR 1810 that I will prepare.